docs/docs-content/clusters/pcg/manage-pcg/pcg-upgrade.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_label: "Upgrade a PCG"
+title: "Upgrade a PCG"
+description: "Learn how to upgrade a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["pcg"]
+---
+
+A PCG upgrade event can be broken down into two categories: Palette agent updates and cluster profile updates. Depending
+on the upgrade event, different steps are required to complete the upgrade.
+
+Palette updates are tied to the Palette platform version. Self-hosted Palette instances must upgrade to the next minor
+or major version to receive the latest PCG agent and cluster profile updates that are compatible with the new version of
+Palette. For example, if you are on Palette 4.1.12, you must upgrade to Palette 4.2.0 or greater to receive the latest
+Palette agent updates that are compatible with Palette 4.2.x.
+
+:::warning
+
+Before upgrading a self-hosted Palette instance to a new minor or major version, allow the PCG agent to automatically
+upgrade to the latest version and manually approve any pending cluster profile updates. This will help you avoid
+compatibility issues between the PCG and the upgraded Palette.
+
+For instance, if you are upgrading from Palette v3.4.x to v4.2.x, ensure both PCG agent and cluster profiles updates are
```

Comment:
This confused me a bit - maybe I am missing some context. When you upgrade Palette from 3.4 to 4.2, do you need to upgrade incrementally? For example, 3.4 -> 4.0 first and then 4.0 -> 4.1 and then 4.1 -> 4.2. It kind of reads like this is the case. 

If so, I think it would help to provide that context here as maybe readers aren't familiar with the Palette. And then clarify what we mean is that the user should make sure that the PCG agent is the most up-to-date with their current minor version